Formula/mingw-w64-binutils.rb

```diff
@@ -0,0 +1,26 @@
+class MingwW64Binutils < Formula
+  desc "Binutils for Windows mingw-w64 (32 and 64 bits)"
+  homepage "https://www.gnu.org/software/binutils/"
+  url "https://ftpmirror.gnu.org/binutils/binutils-2.27.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/binutils/binutils-2.27.tar.gz"
+  sha256 "26253bf0f360ceeba1d9ab6965c57c6a48a01a8343382130d1ed47c468a3094f"
+
+  def install
+    system "./configure", "--disable-werror",
+                          "--target=x86_64-w64-mingw32",
+                          "--enable-targets=x86_64-w64-mingw32,i686-w64-mingw32",
+                          "--prefix=#{prefix}",
+                          "--with-sysroot=#{prefix}"
+    system "make"
+    system "make", "install"
+
+    # Info pages and localization files conflict with native tools
+    info.rmtree
+    (share/"locale").rmtree
```

Comment:
In the spirit of "minor conflicts" (as described in the cookbook), this is what is also done by other developer tool packages in homebrew-core, like gdb (https://github.com/Homebrew/homebrew-core/blob/master/Formula/gdb.rb). I followed the practice.

Another option would be to leave the locale files in the keg, and not symlinking them into `/usr/local/share/locale`. I have not found how to specify that in the recipe.